server.Dockerfile

```diff
@@ -3,8 +3,10 @@ MAINTAINER David Lawrence "david.lawrence@docker.com"
 
 RUN apk add --update git gcc libc-dev && rm -rf /var/cache/apk/*
 
-# Install SQL DB migration tool
-RUN go get github.com/mattes/migrate
+# Pin to the specific v1 version
+RUN git clone -b v1 https://github.com/mattes/migrate.git /go/src/github.com/mattes/migrate/
+RUN go get -u -v github.com/mattes/migrate && \
```

Comment:
Also, if it's already cloned, you might just be able to do a `go install -tags 'mysql' =o ...` and skip the `go get` entirely